game_event/tests.py

```diff
@@ -286,3 +289,46 @@ def test_fail_add_player_to_event_already_in_it(self, client, saved_game_event,
         assert response.status_code == 200
         with pytest.raises(IntegrityError):
             response = client.post(url, {'answer': 'yes'})
+
+@pytest.mark.django_db
+class TestWeatherApi:
+    def test_invalid_city_invalid_weather(self, client, saved_game_event, player):
+        client.force_login(player.user)
+        invalid_city = "wrongcity"
+        url = 'https://api.openweathermap.org/data/2.5/weather'
+        params = {
+            'q': invalid_city,
+            'units': 'metric',
+            'appid': 'bd708048ad9d3ee449660bedad49f30e'
+        }
+        full_url = url + '?' + urllib.parse.urlencode(params)
+        try:
+            response = urllib.request.urlopen(full_url)
+            # assert response.status == 404
```

Comment:
can you please remove this notes?
It implemented in th next block

Reply:
Good point, @amitCohen2 Sure, after we'll fix the broken tests, I'll remove all exceeding prints and note

Reply:
Removed the comments when solving broken tests @amitCohen2 .